streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StreamSinkNode.java

```diff
@@ -60,6 +61,9 @@ public void writeToTopology(final InternalTopologyBuilder topologyBuilder) {
             @SuppressWarnings("unchecked")
             final StreamPartitioner<K, V> windowedPartitioner = (StreamPartitioner<K, V>) new WindowedStreamPartitioner<Object, V>((WindowedSerializer) keySerializer);
             topologyBuilder.addSink(nodeName(), topicNameExtractor, keySerializer, valSerializer, windowedPartitioner, parentNames);
+        } else if (topicNameExtractor instanceof StaticTopicNameExtractor) {
+            final String topicName = ((StaticTopicNameExtractor) topicNameExtractor).topicName;
+            topologyBuilder.addSink(nodeName(), topicName, keySerializer, valSerializer, partitioner, parentNames);
```

Comment:
A hack; otherwise the topologyBuilder doesn't actually track this as an internal-topic sink node ?!?!?!